src/shared/condition.c

```diff
@@ -992,12 +992,12 @@ static int condition_test_psi(Condition *c, char **env) {
         if (r == 1) {
                 pressure_path = path_join("/proc/pressure", pressure_type);
                 if (!pressure_path)
-                        return log_oom();
+                        return -ENOMEM;
```

Comment:
we log at debug level here, so log_oom was wrong too

Reply:
Then, `log_oom_debug()`?

Reply:
sure, done